DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -122,21 +122,22 @@ You can compile and run this Reference Design in the Eclipse* IDE (in Linux*) an
 
  1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
      ```
-     ./mvdr_beamforming.fpga_emu --in=../data         (Linux)
-     ./mvdr_beamforming.fpga_emu.exe --in=../data     (Windows)
+     ./mvdr_beamforming.fpga_emu 1024 ../data .          (Linux)
+     ./mvdr_beamforming.fpga_emu.exe 1024 ../data .      (Windows)
      ```
 
 2. Run the sample on the FPGA device.
      ```
-     ./mvdr_beamforming.fpga --in=../data             (Linux)
+     ./mvdr_beamforming.fpga 1024 ../data .              (Linux)
      ```
 
 ### Application Parameters
 
-| Argument                  | Description
-|---                        |---
-| `--in=<path to input>`    | Specifies the directory that contains the input files (default=`../data`)
-| `--out=<path to output>`  | Specifies the directory to produce output data to (default=`.`)
+| Argument Index        | Description
+|---                    |---
+| 0                     | The number of matrices (default=`1024`)
```

Comment:
I feel like the index should start at 1 instead of 0, since we use argv[1] is the first argument?  Your call though, I don't feel strongly about it.